### ISSUE TITLE
VK_PIPELINE_CACHE_CREATE_READ_ONLY_BIT aliased with a disabled extension

### ISSUE THREAD

**Issue by mackron** [KhronosGroup/Vulkan-Docs#2070]
Not sure if this is intentional, but it seemed strange and it's causing an error in my code generator so thought I'd mention it. Just noticed that `VK_PIPELINE_CACHE_CREATE_READ_ONLY_BIT` if aliased to `VK_PIPELINE_CACHE_CREATE_RESERVED_1_BIT_EXT` which is defined by a disabled extension, "VK_GOOGLE_extension_196". Was this intentional?

```xml
<enum            extends="VkPipelineCacheCreateFlagBits"    name="VK_PIPELINE_CACHE_CREATE_READ_ONLY_BIT" alias="VK_PIPELINE_CACHE_CREATE_RESERVED_1_BIT_EXT"/>
```

Disabled extension XML:
```xml
<extension name="VK_GOOGLE_extension_196" number="196" author="GOOGLE" contact="Jean-Francois Roy @jfroy" supported="disabled">
    <require>
        <enum value="0"                                         name="VK_GOOGLE_EXTENSION_196_SPEC_VERSION"/>
        <enum value="&quot;VK_GOOGLE_extension_196&quot;"       name="VK_GOOGLE_EXTENSION_196_EXTENSION_NAME"/>
        <enum bitpos="1"  extends="VkPipelineCacheCreateFlagBits"
            name="VK_PIPELINE_CACHE_CREATE_RESERVED_1_BIT_EXT"/>
    </require>
</extension>
```

**Comment by oddhack** [comment]
It is a mistake and will be corrected in a few days. The aliasing should be reversed.

**Comment by dgkoch** [comment]
It actually was intentional when Vulkan SC was completely downstream. With the XML merged back upstream, we can finally fix it.

**Comment by oddhack** [comment]
This should be fixed in the 1.3.243 spec update.
